core/src/com/unciv/ui/mapeditor/NewMapScreen.kt

```diff
@@ -0,0 +1,81 @@
+package com.unciv.ui.mapeditor
+
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.scenes.scene2d.ui.ScrollPane
+import com.badlogic.gdx.scenes.scene2d.ui.Table
+import com.unciv.UncivGame
+import com.unciv.logic.map.MapGenerator
+import com.unciv.logic.map.MapParameters
+import com.unciv.logic.map.TileMap
+import com.unciv.models.ruleset.RulesetCache
+import com.unciv.models.translations.tr
+import com.unciv.ui.newgamescreen.MapParametersTable
+import com.unciv.ui.pickerscreens.PickerScreen
+import com.unciv.ui.utils.*
+import kotlin.concurrent.thread
+
+/** New map generation screen */
+class NewMapScreen : PickerScreen() {
+
+    private val mapParameters = MapParameters()
+    private var generatedMap: TileMap? = null
+
+    init {
+        setDefaultCloseAction()
+
+        val newMapScreenOptionsTable = Table(skin).apply {
+            pad(10f)
+            add("Map options".toLabel(fontSize = 24)).colspan(2).row()
+            add(MapParametersTable(mapParameters)).colspan(2).row()
```

Comment:
This is exactly why I separated the map options from the gameplay options in the new game screen =D
If bothrows have 1 element, though, then you don't need the colspan(2)